example/multi-key-handler.sw

```diff
@@ -0,0 +1,20 @@
+// Proof of concept illustrating the possibility of key
+// handlers that process multi-key sequences.
+
+def cons : a * b -> (a -> b) -> (a -> b) = \p. \k. \a.
+  if (a == fst p) {snd p} {k a}
+end
+
+def nil : a -> cmd unit = \a. return () end
```

Comment:
Neat trick for emulating lists.  I will have to borrow this for a challenge scenario.

Reply:
Well, specifically lists-as-append-only-lookup-maps.  This same trick won't work for emulating more general lists.